tests/e2e/tests/pilot/testdata/networking/v1alpha3/sidecar-scope-ns1-ns2.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: sidecar-scope-ns1-ns2
+spec:
+  ingress:
```

Comment:
I guess we could use a single Sidecar without selector for ingress config - with the semantic that if a workload has an inbound port with that number ( and as requested by Louis - it's targetPort, not servicePort ) - we apply the ingress config. 

This will simplify the config for small namespaces - as long as they keep targetPort different for each service ( which will be required for mode=NONE and should probably be recommended)